src/Inkluzitron/Data/BotDatabaseContext.cs

```diff
@@ -36,6 +36,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             {
                 builder.Property(o => o.Gender).HasConversion<string>();
                 builder.HasIndex(o => o.KisNickname).IsUnique();
+                builder.Ignore(o => o.BirthdayDate);
```

Comment:
Místo fluent API bych použil `[NotMapped]` atribut. Sice to funguje stejně, ale bude přímo v entitě známo, že daná položka není mapovaná do databáze.

Reply:
Nakonec to není potřeba dělat přes tento builder, ale dávalo mi smysl to sem dát, aby se nemíchaly atributy a fluent API.